web/styles/dark_theme.css

```diff
@@ -515,17 +515,34 @@
         border-color: hsl(0deg 0% 0% / 60%);
     }
 
+    .emoji-picker-popover .emoji-showcase-container,
+    .emoji-picker-popover .emoji-popover .emoji-popover-category-tabs,
+    .emoji-picker-popover .emoji-popover .emoji-popover-top {
+        background-color: hsl(0deg 0% 0% / 20%);
+    }
+
     .emoji-picker-popover
         .emoji-popover
         .emoji-popover-category-tabs
         .emoji-popover-tab-item.active {
         background-color: hsl(0deg 0% 0% / 50%);
     }
 
+    .emoji-popover-emoji {
+        &:focus {
+            background-color: hsl(212deg 28% 8% / 75%);
+            box-shadow: 0 0 1px hsl(0deg 0% 98%);
+        }
+
+        &.reacted {
+            background-color: hsl(0deg 0% 0% / 50%);
+            border-color: hsl(0deg 0% 0% / 90%);
+        }
+    }
```

Comment:
This isn't new with this PR, but this is the focus+reacted behavior that I see in the emoji picker for reactions:

![image](https://github.com/zulip/zulip/assets/2746074/b6a364d6-2099-4685-9223-64c4ad57f976)

That's probably a bug we should fix in a follow-up?